tests/Functional/WorkflowCommandsTest.php

```diff
@@ -72,12 +74,9 @@ public function testWorkflowOperationsAndLogsCommands()
         $this->assertIsArray($testOperation);
         $this->assertNotEmpty($testOperation);
         $this->assertArrayHasKey('type', $testOperation);
-        $this->assertEquals('quicksilver', $testOperation['type']);
         $this->assertArrayHasKey('result', $testOperation);
-        $this->assertEquals('succeeded', $testOperation['result']);
```

Comment:
These tests should be restored

Reply:
Rebase with the default branch, the tests are fixed there.